model/function.go

```diff
@@ -29,6 +29,8 @@ const (
 	FunctionTypeAsyncAPI FunctionType = "asyncapi"
 	// FunctionTypeOData ...
 	FunctionTypeOData FunctionType = "odata"
+	// FunctionTypeCustom ...
```

Comment:
we should complete the comment, such as `FunctionTypeCustom is the additional function types that extend the ones defined in the specification which implement by runtime`

Reply:
@spolti please add the description from the scheme.